charts/enterprise-metrics/CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## Unreleased
 
+## 1.4.5
+
+* [CHANGE] Enabled enterprise authentication by default.
+  > **Breaking:** This change can cause loosing accesss to the GEM cluster in case auth.type has not
+  > been set explicitly.
+  > This is a security related change and therefore released as bugfix.
```

Comment:
I understand this to mean that we are incrementing the patch version rather than a minor version despite it being a breaking change although I was initially confused by the word "bugfix" which is one of our CHANGELOG categories.

Perhaps:

```suggestion
  > This is a security related change and therefore is in a patch release.
```

^ I'm struggling with the right wording too.